### PR TITLE
docker: minimise number of image layers

### PR DESCRIPTION
**Pull request by hachreak**
* Modifies the Dockerfile in order to apply best practices.

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

**Comment by hachreak** [comment]
See https://docs.docker.com/articles/dockerfile_best-practices/#run

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
 WORKDIR /code
-RUN apt-get update
-RUN apt-get install -y git software-properties-common python python-dev python-pip
-RUN add-apt-repository -y ppa:chris-lea/node.js
-RUN apt-get update
-RUN apt-get install -y nodejs
-RUN npm install bower
-RUN pip install -r requirements.txt
+ENV DEBIAN_FRONTEND noninteractive
+RUN add-apt-repository -y ppa:chris-lea/node.js && \
```
Comment (tiborsimko) [inline Dockerfile]: Yes, it is good to join lines to minimise the number of Docker image levels, but `add-apt-repository` does not exist if you don't do `update` first...

Comment (tiborsimko) [inline Dockerfile]: ... also, you had some `-y` missing and `pip` was not installing.  I had fixed all those and I'll merge.

**Comment by tiborsimko** [comment]
Thanks, corrected as mentioned, and merged as cd534e3.

**Pull request closed without merging**